File: 3174-minimum-number-of-changes-to-make-binary-string-beautiful/3174-minimum-number-of-changes-to-make-binary-string-beautiful.cpp
class Solution {
public:
    int minChanges(string s) {
        // return 0;
        int one=0,zero=0;
        // for(auto x:s){
        //     if(x=='1')one++;
        //     else zero++;
        // }
        // if(one<=zero)return one;
        // return zero;
        int cnt=0;
        for(int i=0;i<s.length()-1;i+=2){
            if(s[i]!=s[i+1])cnt++;
        }
        return cnt;
    }
};

File: 3174-minimum-number-of-changes-to-make-binary-string-beautiful/3174-minimum-number-of-changes-to-make-binary-string-beautiful.js
/**
 * @param {string} s
 * @return {number}
 */
var minChanges = function(s) {
    let cnt=0;
    for(let i=0;i<s.length;i=i+2){
        if(s[i]!=s[i+1])cnt++;
    }
    return cnt;
};